mongo/mongo.go

```diff
@@ -435,26 +441,38 @@ func EnsureServer(args EnsureServerParams) (Version, error) {
 
 func ensureServer(args EnsureServerParams, mongoKernelTweaks map[string]string) (Version, error) {
 	var zeroVersion Version
+	var err error
```

Comment:
Can't see where this is used. In any case, if needed, it needs to be declared next to where it's used.